### ISSUE TITLE
Cursor blink doesn't work when using WebGL renderer

### ISSUE THREAD

**Issue by slammayjammay** [zeit/hyper#3626]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS Mojave 10.14
- **Hyper.app version**: 3.0.0
- **The issue is reproducible in vanilla Hyper.app**: Yes

## Issue
Not a duplicate, but might be related to #3533, #3041.

Cursor doesn't blink when using WebGL renderer, but does when using canvas renderer.

Hyper config:
```js
module.exports = {
  config: {
    cursorBlink: true
  }
};
```
I tested canvas rendering by adding the line `backgroundColor: 'rgba(0, 0, 0, 0.9)'`:
```
WebGL Renderer has been disabled since it does not support transparent backgrounds yet. Falling back to canvas-based rendering.
```

**Comment by pcnate** [comment]
looks like I have the same. Hadn't noticed it yet

**Comment by ppot** [comment]
Yeah I know. I'm slowly rewriting the whole application front. But faced this problem since webgl. Don't know why. Even with no plugins installed and all even't hooks removed form the code. Still a mistery as of why.

**Comment by evxn** [comment]
Maybe related to #3664

**Comment by virajkanwade** [comment]
Similar behavior on MacOS

**Comment by Pikdo** [comment]
I have the same error cursorBlink setting doesn´t work on hyper last version, +3.0.2 with "zsh" and "oh my myszh "installed on win10 WSL Ubuntu 20.04, the cursor is static.

**Comment by bpeake-illuscio** [comment]
Same here on OSX + oh-my-zsh. Weirdly, changing the zoom level fixes it for a given tab.

**Comment by MorganFinTW** [comment]
> Same here on OSX + oh-my-zsh. Weirdly, changing the zoom level fixes it for a given tab.

same here on Windows 10

**Comment by carlos3g** [comment]
Same here on Fedora 33
